Arbiter: Handle GPS week rollover

Tipo874 opened the pull request:
This patch handles the GPS week rollover that happened in 2019 for old firmware
in Arbiter GPS clocks.

Tested on an Arbiter 1092B.

Signed-off-by: Thomas Tornblom <thomas@hax.se>

he32 commented:
Bah, this is against the wrong branch.  Closing this and trying again
against the correct branch, hopefully...

Tipo874 committed:
Arbiter: Handle GPS week rollover

This patch handles the GPS week rollover that happened in 2019 for old firmware
in Arbiter GPS clocks.

Tested on an Arbiter 1092B.

Signed-off-by: Thomas Tornblom <thomas@hax.se>
